mappings/none/BlockPattern.mapping

```diff
@@ -0,0 +1,58 @@
+CLASS none/asq none/BlockPattern
+	CLASS none/asq$a BlockStateCacheLoader
+		FIELD a world Lnone/aiw;
+		METHOD <init> (Lnone/aiw;Z)V
+			ARG 0 world
+		METHOD a load (Lnone/cn;)Lnone/asp;
+			ARG 0 pos
+	CLASS none/asq$b Result
+		FIELD a frontTopLeft Lnone/cn;
+		FIELD b forwards Lnone/cu;
+		FIELD c up Lnone/cu;
+		FIELD d cache Lcom/google/common/cache/LoadingCache;
+		FIELD e leftSize I
```

Comment:
It would be possible to change the name of leftSize / downSize / forwardSize to width, height, and depth? Wasn't too sure if forwards should be depth? or Breadth? however, so left it as it is for the moment while better names could be considered.